playbook/app/pb_kits/playbook/pb_textarea/_textarea.tsx

```diff
@@ -52,13 +52,13 @@ const Textarea = ({
   label,
   maxCharacters,
   name,
-  onChange = () => {},
```

Comment:
I ended up changing this back to a default empty callback, and just added `eslint-disable-next-line` above it.

I think the alternative is to have an empty callback that does something, like 

```js
onChange = (value) => {
  console.log(value)
},
```

Or, in all the Jest test files add an empty callback to each `textarea` component, otherwise you get this error:

```Warning: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`.```